bite-size-notes/week-8-c-LQG-stability-smvial.ipynb

```diff
@@ -0,0 +1,363 @@
+    "\n",
+    "K = f \\begin{bmatrix}\n",
+    "1 & 1 \n",
+    "\\end{bmatrix} \\qquad \\text{ and } \\qquad L = d\\begin{bmatrix}\n",
+    "1 \\\\ \n",
+    "1 \n",
+    "\\end{bmatrix}\n",
+    "\n",
+    "\n",
+    "$$\n",
+    "\n",
+    "\n",
+    "Putting all of this together, our full system dynamics become:\n",
+    "$$\n",
+    "\n",
+    "\\begin{bmatrix}\n",
+    "A & BK\\\\\n",
+    "LC & A-LC-BK \n",
+    "\\end{bmatrix} \n",
+    "\\begin{bmatrix}\n",
+    "x \\\\ \n",
+    "\\hat{x} \n",
+    "\\end{bmatrix} = \n",
+    "\\begin{bmatrix}\n",
+    "\\dot{x} \\\\ \n",
+    "\\dot{\\hat{x}} \n",
+    "\\end{bmatrix}\n",
+    "\n",
+    "$$\n",
+    "\n",
+    "Here $\\hat{x}$ is our extimate of $x$ from observation. This system is comes from these a synthesis of the following equations:\n",
+    "$$\n",
+    "\n",
+    "\\begin{align*}\n",
+    "\\dot{x} &= Ax + Bu & \\qquad u &= -K\\hat{x} \\\\\n",
+    "\\dot{\\hat{x}} &= A\\hat{x} + Bu + L(y-\\hat{y}) & \\qquad y &= Cx \\\\\n",
+    " & & \\qquad \\hat{y} &= C\\hat{x} \\\\\n",
+    "\\end{align*}\n",
+    "\n",
+    "$$\n",
+    "\n",
+    "The last step in constructing this LQG system is to say that the closed loop controller has a nominal scalar gain, $m$. That brings our full system matrix to:\n",
+    "$$\n",
+    "\\begin{bmatrix}\n",
+    "A & mBK\\\\\n",
+    "LC & A-LC-BK \n",
+    "\\end{bmatrix} =\n",
+    "\\begin{bmatrix}\n",
+    "1 & 1 & 0 & 0\\\\\n",
+    "0 & 1 & -mf & -mf\\\\\n",
+    "d & 1 & 1-d & 1\\\\\n",
+    "d & 0 & -d-f & 1-f\\\\\n",
+    "\\end{bmatrix} \n",
+    "$$"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "##### Exposing Instability\n",
+    "\n",
+    "Recall that in dynamic systems, the primary requirment is that the eigenvalues of the system dynamics all be less than zero.  With that in mind, consider the charactaristic equation of our system matrix.\n",
+    "$$\n",
+    "\\lambda^4 + (d + f - 4)\\lambda^3 + (df - 2(d+f) + b)\\lambda^2 + (d+f - 4 + 2df(m-1))\\lambda +(1 + (1-m)df) = 0\n",
+    "$$\n",
+    "\n",
+    "Focus on the final two terms (the only terms containing $m$).  Note that in order for all of the eigenvalues to be less than 0, these last two terms must be greater than 0.  \n",
+    "$$\n",
+    "\n",
+    "\\begin{align*}\n",
+    "1 + (1-m)df &> 0 & \\qquad d+f - 4 + 2df(m-1) &> 0 \\\\\n",
+    "m &< 1 + \\frac{1}{df}  & \\qquad m &> \\frac{4 - d - f}{2df} + 1  \\\\\n",
+    "\\end{align*}\n",
+    "\n",
+    "$$\n",
+    "\n",
+    "When d and f (or equivalently, q and $\\sigma_w$) become sufficiently large, $ m \\gtrsim 1$ AND $m \\lesssim 1$.  Since, $g$ and $f$ cannot actually go to infinity, the system will technically be stable as long as it exhibits a gain of exactly one.  However, if it deviates slightly in either direction, it will become unstable! There is the rub: **An LQG system has no guarunteed stability margin.  It may operate arbitrarily close to a pole.** \n",
```

Comment:
I don't know if i understand what is meant by operating arbitrarily close to a pole. Perhaps you meant close to an unstable pole?

Reply:
This is my fault.  I should have been more careful using terminology from robust control theory when I have not yet learned it.  I understand that in this field, we refer to unstable fixed points as poles.  That was not what I should have said.  What I should have said is that the system dynamics can operate arbitrarily close to having positive eigenvalues - and thus becoming unstable. 

I have changed the language accordingly.  Good editorial input.